marathon/check.py

```diff
@@ -74,6 +74,11 @@ def check(self, instance):
         if response is not None:
             self.gauge('marathon.deployments', len(response), tags=instance_tags)
 
+        # Number of queued applications
+        response = self.get_json(urljoin(url, "v2/queue"), timeout, auth, acs_url, ssl_verify)
+        if response is not None:
+            self.gauge('marathon.queue', len(response['queue']), tags=instance_tags)
```

Comment:
I see there are other info in the response payload, I'd name the metric `marathon.queue.size` so we leave room for future additions.